mcs/resource_mcs_db_instance.go

```diff
@@ -27,14 +27,19 @@ const (
 )
 
 type dbInstanceStatus string
+type dbCapabilityStatus string
 
 var (
```

Comment:
i guess it's const

Reply:
I think we can write dbCapabilityStatus in the same vein as dbInstanceStatus and clusterStatus